src/dorsal.js

```diff
@@ -226,12 +226,16 @@ DorsalCore.prototype._wireElements = function(nodes, plugins, deferred) {
  */
 DorsalCore.prototype._wireElement = function(el, plugins, deferred) {
     var self = this,
+        dfd = new DorsalDeferred(),
         log = new DorsalLog(this.DEBUG);
 
+    deferred.when(dfd);
```

Comment:
actually we are passing the original `deferred`. 

this function seemed the best candidate to hold the creation of each deferred given that is the end of the chain (is in where that we create the `window.setTimeout` function.

If not, we would need chain to return the new deferred and store it in every existent entry point (this is `wireElements` , `wireElementFrom`  and `wireElement`.